bin/build-plugin-zip.sh

```diff
@@ -98,6 +98,7 @@ zip -r gutenberg.zip \
 	packages/block-serialization-default-parser/*.php \
 	post-content.php \
 	$build_files \
+	build-module \
```

Comment:
I don't remember why there was pattern matching used for files in the `build` folder. @youknowriad, is there anything important we might miss when using `build-module`? At the moment, we want all files to be included from there.

Reply:
I created related PR https://github.com/WordPress/gutenberg/pull/65064